config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-presubmits.yaml

```diff
@@ -8,7 +8,6 @@ presubmits:
     - master
     # NON-BLOCKING; only triggered manually, use this for debug test script
     always_run: false
-    skip_report: true
```

Comment:
this needs to be marked `optional: true` now.

Reply:
I think that should be ok.. we're only going to trigger it on WIP or do-not-merge PRs.
Doesn't hurt even if it's merge-blocking.

Reply:
if it gets marked as merge blocking / shows up as skipped, we get a lot of inbound pings about "what is this new job" :-)